tests/shared/src/test/scala/scala/meta/tests/quasiquotes/ErrorSuite.scala

```diff
@@ -803,23 +803,6 @@ class ErrorSuite extends FunSuite {
     )
   }
 
-  test("triple-dot mixes with something else in parameter lists") {
```

Comment:
Why did you remove the test? We no longer have this limitation?

Reply:
because it's not broken anymore (it's capable of parsing this quasiquote now). i added a test in success suite instead.